W-12168204-permissions (#76)
mulesoft/docs-hosting: cloudhub-2/modules/ROOT/pages/ps-assign-permissions.adoc

* To set up and manage a private space, you need the CloudHub Network Administrator permission.
* To view a private space, you need the CloudHub Network Viewer permission.

> * To deploy applications from Exchange, you must have the Exchange Contributor and Exchange Viewer permissions.

Author: ```suggestion
* To deploy applications from Exchange, you need the Exchange Contributor and Exchange Viewer permissions.
```

* To deploy applications in Runtime Manager, you need the Create Applications permission.

> * To view all applications in the *Applications* page in Runtime Manager, you need the Read Applications and Read Runtime Fabric permissions.

Reviewer: ```suggestion
* To view all applications on the *Applications* page in Runtime Manager, you need the Read Applications and Read Runtime Fabric permissions.
```
